src/accounts/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+
+app_name = 'accounts'  # pylint: disable=invalid-name
```

Comment:
Do we have to have this lowercase for Django (and using the pylint disable) instead of using the uppercase for constants?

Reply:
I in fact see it is a django requirement! Thanks - all good on this comment